deploy/BUILD.bazel

```diff
@@ -90,6 +90,14 @@ java_export(
     ],
 )
 
+sh_test(
```

Comment:
Shouldn't this be used somewhere? I can not find any call sites.
A local run exits with code `126` and this message (when printing it):
```
/[...]/jazzer_version_test: line 37: /[...]/execroot/jazzer/bazel-out/k8-opt/bin/deploy/jazzer_version_test.runfiles/jazzer/deploy/jazzer-project.jar: cannot execute binary file: Exec format error
```

Reply:
I pushed a new version, could you test this again?